modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerTest.java

```diff
@@ -156,30 +157,36 @@ public void testCreateZoneIfExists() throws Exception {
                 new DistributionZoneConfigurationParameters.Builder(ZONE_NAME).dataNodesAutoAdjust(100).build()
         ).get(5, TimeUnit.SECONDS);
 
+        CompletableFuture<Void> fut;
+
+        fut = distributionZoneManager.createZone(
+                new Builder(ZONE_NAME).dataNodesAutoAdjust(100).build()
+        );
+
         try {
-            distributionZoneManager.createZone(
-                    new DistributionZoneConfigurationParameters.Builder(ZONE_NAME).dataNodesAutoAdjust(100).build()
-            ).get(5, TimeUnit.SECONDS);
+            fut.get(5, TimeUnit.SECONDS);
         } catch (Exception e0) {
             e = e0;
         }
 
         assertTrue(e != null);
-        assertTrue(e.getCause().getCause() instanceof DistributionZoneAlreadyExistsException, e.toString());
+        assertTrue(e.getCause() instanceof DistributionZoneAlreadyExistsException, e.toString());
```

Comment:
I like the approach that always requires a message, it looks a lot friendlier.
```
    assertTrue("Expected exception was not thrown.", e != null);
    assertTrue("Unexpected type of exception (requires DistributionZoneAlreadyExistsException)", e.getCause() instanceof DistributionZoneAlreadyExistsException);
```

Reply:
Makes sense. Fixed.